tests/org.eclipse.passage.lic.json.tests/src/org/eclipse/passage/lic/json/tests/CertificateTransportTest.java

```diff
@@ -35,13 +33,12 @@
 public class CertificateTransportTest {
 
 	@Test
-	@Ignore // FIXME #566331
 	public void examinationNotPassed() {
 		try {
 			ZonedDateTime time = ZonedDateTime.now();
 			CertificateTestData data = new CertificateTestData();
-			String serialized = mapper().writeValueAsString(new AcquiredExaminationCertificate(data.permissions(),
-					data.restrictions(), DateTimeFormatter.ISO_ZONED_DATE_TIME.format(time)));
+			String serialized = mapper().writeValueAsString(new AcquiredExaminationCertificate(
+					Map.of(data.requirement(), data.permission()), data.restrictions(), time));
```

Comment:
Technically we build with j11, but cannot afford using 8+api for now due to support obligations.

`Map::of` looks like something younger.